### PR TITLE
Don't close logger before initializing, fix check for yum-cron.service

### PR DESCRIPTION
**Pull request by adjackura**

### CODE REVIEW

File: ospatch/system_linux.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ func disableAutoUpdates() {
 		if err != nil {
 			if eerr, ok := err.(*exec.ExitError); ok {
 				// Error code of 1 indicates disabled.
```
Comment (ryanwe) [inline ospatch/system_linux.go]: Does the comment mean "was already disabled" or "was successfully disabled" or both?

Comment (adjackura) [reply ospatch/system_linux.go]: Is currently disabled, the above check is just running "is-enabled"

**Pull request merged**